nailgun/entities.py

```diff
@@ -3358,13 +3360,6 @@ def create(self, create_missing=None):
             id=self.create_json(create_missing)['id'],
         ).read()
 
-    def update_payload(self, fields=None):
```

Comment:
Is this conversion from `system_uuids` to `system_ids` no longer necessary?

Reply:
Since neither of those parameters is mentioned in 6.8's APIdoc, I suppose it's obsolete. (But I'm not the author of that workaround so I can't be absolutely sure.)